charts/tenantfilesystem/README.md

```diff
@@ -0,0 +1,81 @@
+# tenantfilesystem
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+
+Helm Chart for installing a custom tenantfilesystem xrd
+
+## Installation
+Installs custom wadtfy tenantfilesystem crossplane XRD on an existing Kubernetes cluster for a specific namespace.
+
+**Preconditions**:
+* [Crossplane](https://crossplane.io) need to be installed on Kubernetes.
+* The default [AWS Crossplane Provider](https://github.com/crossplane-contrib/provider-aws) is used for provisioning AWS Resources.
+
+### Add Helm repository
+
+```shell
+helm repo add wadtfy-custom-component https://dvpe-cloud.github.io/wadtfy-custom-components
+helm repo update
+```
+
+## Install wadtfy tenantfilesystem chart
+
+```sh
+helm install tenantfilesystem . -f values.yaml
+```
+
+## Use TenantFileSystemClaim to create Filesystems
+
+Create a `tenantfilesystemclaim.yaml` and set the Tenant Account related values
+
+Sample:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: tenantfilesystemclaim
+metadata:
+  name: tenantname-filesystemname
+  namespace: tenant-namespace
+spec:
+  compositionRef:
+    name: tenantfilesystem
+  # Tenant AWS Account ID
+  tenantAccountID: ""
+  # Crossplane ProviderConfig from Tenant
+  tenantProviderConfig:
+  # AWS VPC ID
+  vpcId:
+  # AWS VPC Subnet ID's
+  vpcSubnetIdA:
+  vpcSubnetIdB:
+  vpcSubnetIdC:
+  # AWS Account Region
+  region:
+  # EFS Performance Mode
+  efsPerformanceMode:
+  # EFS Throughput Mode
+  efsThroughputMode:
+  # StorageClass specific values
+  storageClass:
+    directoryPerms: "700"
+    basePath: "/dynamic_provisioning"
+  writeConnectionSecretToRef:
+    name:
+```
```

Comment:
Why not define the API like:
```
tenant:
   accountId:
   providerConfig:
vpc:
   id:
   subnetIdA:
   subnetIdB:
   subnetIdC:
...
```
Just a suggestion. What do you think @mithie?

Reply:
because those are the vpc settings of the tenant and therefore should be specified in the context of the tenant as well. if you want to do it that way it should then be

```
tenant:
   accountId:
   providerConfig:
   vpc:
      id:
      subnetIdA:
      subnetIdB:
      subnetIdC:
```

Reply:
I like that, it is way clearer that a remote vpc is meant, which wasn't that obvious before